frontend/src/lib/components/MemberSelect.tsx

```diff
@@ -9,11 +9,12 @@ import { UserBasicType } from '~/types'
 export type MemberSelectProps = {
     defaultLabel?: string
     // NOTE: Trying to cover a lot of different cases - if string we assume uuid, if number we assume id
-    value: UserBasicType | string | number | null
+    value: string | number | null
```

Comment:
why no longer also userbasictype?

Reply:
Looked at all the usages around the codebase and it was only ever a `string | number | null`. Thought it best to remove the extra complexity if no one was using it